engines/docker/generated_configschema.go

```diff
@@ -0,0 +1,47 @@
+// This source code file is AUTO-GENERATED by github.com/taskcluster/jsonschema2go
+
+package docker
+
+import "github.com/taskcluster/taskcluster-worker/runtime"
+
+type (
+	// Config applicable to docker engine
+	Config struct {
+
+		// Root Volume blah blah
+		RootVolume string `json:"rootVolume"`
+	}
+)
+
+func ConfigSchema() runtime.CompositeSchema {
```

Comment:
Are you sure we don't want to export them - I think it wouldn't hurt if someone using a package could see the json schema, as this helps when we write automated tooling to e.g. publish schemas etc. Of course, if we really don't want it, we can - it feels though correct that we should be able to generically infer stuff about plugins/engines etc. That said, the generated code is not for the engine/plugin itself, but just package identifiers, so yeah, I guess we don't want to export that. Fair enough. Yes, I can add support for that.